docs/bank-accounts/Basics.md

```diff
@@ -40,6 +40,9 @@ WealthKernel perform checks on bank accounts to makes sure that the bank account
 
 In the rare cases where we cannot automatically verify a bank account belongs to the party, we may need additional information to prove that the bank account belongs to them. The bank account status will remain in a `Pending` state until the additional information has been provided and verification has been successful. To receive the most up-to-date information about a bank account's status, we recommend using the bank account webhooks which are detailed below.
 
+<!-- theme: info -->
+> Please note that we cannot perform automatic checks for international bank accounts with BIC and IBAN.
```

Comment:
I'd expand this a little further and mention that they will need to be manually checked by our compliance team, and end the notice with something like "Contact your tenant manager for more information."

Reply:
Also, may be worth adding some information about our validation for IBAN and BIC as mentioned in the API documentation

Reply:
Do you mean the modulus check validation? or what validation do we mention in the API docs?

Reply:
I found your comment, it was kind of buried under the other comments. I'll add something